tests/unit/kvmagent/test_volume_cache_lvm_filter.py

```
@@ -0,0 +1,77 @@
+import pytest
+
+try:
+    ModuleNotFoundError
```

Ruff (0.15.20)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)